examples/htlc-pyteal-ts/scripts/withdraw/htlc-withdraw.ts

```diff
@@ -7,21 +7,22 @@
  * if the seller is able to provide the secret value that corresponds to the hash in the program.
 */
 import { stringToBytes } from "@algorand-builder/algob";
-import { AlgobDeployer, AlgobRuntimeEnv } from "@algorand-builder/algob/src/types";
-import { AlgoTransferParam, SignType, TransactionType } from "@algorand-builder/runtime/build/types";
+import * as algob from "@algorand-builder/algob";
+import * as runtime from "@algorand-builder/runtime";
 
 import { executeTx, prepareParameters } from "./common";
 
-async function run (runtimeEnv: AlgobRuntimeEnv, deployer: AlgobDeployer): Promise<void> {
+async function run (
+  runtimeEnv: algob.types.AlgobRuntimeEnv, deployer: algob.types.AlgobDeployer): Promise<void> {
   const { alice, scTmplParams, secret } = prepareParameters(deployer);
   const wrongSecret = 'hero wisdom red split loop element vote belt';
 
   let lsig = await deployer.loadLogic('htlc.py', [stringToBytes(wrongSecret)], scTmplParams);
   let sender = lsig.address();
 
-  const txnParams: AlgoTransferParam = {
-    type: TransactionType.TransferAlgo,
-    sign: SignType.LogicSignature,
+  const txnParams: runtime.types.AlgoTransferParam = {
+    type: runtime.types.TransactionType.TransferAlgo,
```

Comment:
Is there a reason in `ts` files we are using `runtime.types` and in other js examples we are using only types? BDW, i think using `runtime.types` is more intuitive. However, to avoid long names, can we do:
```
import * as runtime from "@algorand-builder/runtime";
const { TransactionType, SignType } = runtime.types;
...
use types below..

```

Reply:
we can use runtime.types in js files also, but there are only two imports that only from runtime, so I think types. is better

I am not sure about the second point, in other files we have algob and runtime, so runtime.types is more intuitive